src/Makefile

```diff
@@ -149,4 +150,4 @@ DRIVER_PATH =
 ## include the makefile-engine
 DEVEL_DIRECTORY := \
 	$(shell findpaths -r "makefile_engine" B_FIND_PATH_DEVELOP_DIRECTORY)
-include $(DEVEL_DIRECTORY)/etc/makefile-engine
+include $(DEVEL_DIRECTORY)/etc/makefile-engine 
```

Comment:
Won't build with the trailing space

Reply:
I always have a random trailing whitespace :/

Reply:
You can configure Pe to remove trailing whitespaces under Preferences - Files.

Reply:
Wait, really? How come I didn't know about this feature sooner :(